src/test/java/GitHubTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.elementToBeClickable;
+
+public class GitHubTest extends BaseTest {
+
+    @Test
+    public void verifyThatLogioOnTheLoginPageIsDisplayed() {
+        HomePage homePage = new HomePage(driver);
+        Assertions.assertTrue(homePage.goToLoginPage().getLogo().isDisplayed());
+    }
+
+    @Test
+    public void verifyLoginIsseccessful() {
+        HomePage homePage = new HomePage(driver);
+        Assertions.assertTrue(homePage.goToLoginPage().loginSuccessful("nadia.suworowa@hotmail.com",
+                        "Cologne123!!!")
+                .getLogoOnTheMainPage().isDisplayed());
+    }
+
+    @Disabled
+    @Test
+    public void verifyFailedLogin() {
+        HomePage homePage = new HomePage(driver);
+        homePage.goToLoginPage().negativeLogin("uhdfurbj", "sedjvodkfb")
+                .validateErrorMessage("Incorrect username or password.");
+    }
+
+    @Test
+    public void testCreateNewIssue() {
```

Comment:
Your test steps start at once from the Issues page. Steps should start from Home page, login, go to some Repository, and only from some Repository we can create a new issue